### PR TITLE
Fix: User properties submenu rendering based on position

### PR DESCRIPTION
**Pull request by helenphan0**

### CODE REVIEW

File: src/components/DropDownMenu.js [hunk 1]
```diff
@@ -65,7 +65,7 @@ export default class DropDownMenu extends React.Component {
 		        top: '100%',
 		        right: 0,
 		        backgroundColor: '#eee',
-		        maxHeight: 370,
+		        maxHeight: 180,
```
Comment (GuruM) [inline src/components/DropDownMenu.js]: should probably be a constant at the top of the file now

**Comment by GuruM** [comment]
why are we doing this again

**Comment by helenphan0** [comment]
With the submenu in User Properties, if you're on a slideout positioned along the top, you can't see the first couple of values or access the searchbar. Aside from that, the menu itself is quite long. Since we have a scrollbar on it, it shouldn't be too much longer than the WYSIWYG itself.

**Comment by GuruM** [comment]
Hmm, it seems like a larger problem is that the user property menu dropdown appears on top rather than underneath. If you create a button in the same place, the button's action's dropdown menu appears on the bottom, which is correct. 180px seems really short for a dropdown menu, so maybe fixing the user property menu's dropdown to appear in the right place takes care of the need to change the menu height

**Comment by GuruM** [comment]
You're right, 370px is probably too big... A good benchmark might be fitting all the button actions without scrolling:
image

so 300px?

**Comment by GuruM** [comment]
The rating limit dropdown menu probably shouldn't be a dropdown menu either 🤔

**Pull request merged**